code/index.html

```diff
@@ -8,5 +8,11 @@
   </head>
   <body>
     <h1>This is your Pizza Bot</h1>
+
+    <p id="pizzaConfirmation"></p>
```

Comment:
For id's and classes we should rather use `kebab-case` than `camelCase` names, so
`pizza-confirmation` rather than `pizzaConfirmation`